docs/make.jl

```diff
@@ -13,6 +13,7 @@ makedocs(
                   "rational.md",
                   "modn.md",
                   "modp.md",
+		            "transExt.md",
```

Comment:
Looks like spacing is wrong here. Probably you used tabs instead of spaces or vice versa.

Reply:
Done